Maple2.Database/Storage/Game/GameStorage.ServerInfo.cs

```diff
@@ -33,6 +33,7 @@ public void DailyReset() {
                 Context.Database.ExecuteSqlRaw("UPDATE `account` SET `PremiumRewardsClaimed` = DEFAULT");
                 Context.Database.ExecuteSqlRaw("UPDATE `character-config` SET `GatheringCounts` = DEFAULT");
                 Context.Database.ExecuteSqlRaw("UPDATE `nurturing` SET `PlayedBy` = '[]'");
+                Context.Database.ExecuteSqlRaw("UPDATE `home` SET `DecorationRewardTimestamp` = 0");
```

Comment:
Shouldn't we also be resetting InteriorRewardsClaimed?

Reply:
InteriorRewardsClaimed is the progression to level 10 in life skills